managers/asset-operations-manager.js

```diff
@@ -102,6 +102,24 @@ class AssetOperationsManager {
         };
     }
 
+    async getCurrentAllowance(options = {}) {
```

Comment:
Open for suggestions on naming, didn't want to call it getAllowance as it can be misunderstood by users